tests/spec/core/highlight.html

```diff
@@ -3,8 +3,7 @@
   <head>
     <meta charset='utf-8'>
     <title>Replace me with a real title</title>
-    <script class='remove' src='/js/deps/require.js' data-main='/js/profile-w3c-common'>
-    </script>
+    <script class='remove' src='/builds/respec-w3c-common.js'></script>
```

Comment:
I'm worried about this one, because it means we need to pre-build before testing, which we don't normally do. We might need to come up with an alternative solution.

Reply:
Actually, ignore me... this only applies to Karma, which on Travis is built so we are fine.

Reply:
>it means we need to pre-build before testing

I think you already are required to pre-build, otherwise even the requirejs build won't get the latest code.

That said, this change wasn't to be included here... I'll roll this back.

Reply:
>Actually, ignore me... this only applies to Karma, which on Travis is built so we are fine.

Oh, okay then 😄